src/libs/tmdk/src/commands/init.ts

```diff
@@ -17,51 +14,57 @@ import path from "path";
 import {
   ListComponentTypesCommandOutput,
   ListScenesCommandOutput,
+  ListEntitiesCommandOutput,
+  GetEntityCommandOutput,
 } from "@aws-sdk/client-iottwinmaker";
 
 export type Options = {
-  region: string | undefined;
-  "workspace-id": string | undefined;
-  out: string | undefined;
+  region: string;
+  "workspace-id": string;
+  out: string;
+};
+
+export type tmdk_config_file = {
+  version: string;
+  component_types: string[];
```

Comment:
If we change the name here we also need to change the name used in unit tests and other code, please carefully check the usage of `component-types` before resolving this comment

Reply:
Yes, unfortunately it seemed that I was not able to use a dash in the name for a type, so I had to change to underline. I only updated all instances in these 2 files, I'll look into testing code

Reply:
You can use a name in the type if you put it in quotation marks, although I think the underscore is better stylistically. Up to you to revert to `component-types` or update all references to `component_types`